components/Header/Header.js

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+
+function Header() {
+  return (
+    <Background>
+      <StyledHeadline>Capstone Project</StyledHeadline>
+    </Background>
+  );
+}
+
+const Background = styled.div`
+  background-color: lightgreen;
+  padding: 20px;
```

Comment:
You could think about using relative units like em for padding, margin, border-radius, etc. This would make your components scale proportionally with their font-size.

Reply:
i did not think about that one, thanks :)